packages/make-styles-webpack-loader/config/api-extractor.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You can probably get rid of API Extractor for this package. It's less relevant when the package just exports an implementation of a standard webpack loader pattern, as opposed to an API people will directly use (if I'm understanding right).

Reply:
I had a similar discussion with @Hotell in https://github.com/microsoft/fluentui/pull/18719#issuecomment-868471909, conclusion was that it will be discussed on Build team sync (https://github.com/microsoft/fluentui/pull/18719#issuecomment-869702986). I will keep it for consistency for now with other migrated packages.

Reply:
thanks @layershifter 🙌